Food_Tracker_App/app/src/main/res/layout/ingredient.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/ingredientDescription"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="20dp"
+        android:paddingLeft="10dp"
+        android:text="Description"
+        android:textSize="20dp" />
+
+    <RelativeLayout
+        android:id="@+id/expand_ingredients"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+
+        <TextView
+            android:id="@+id/ingredientExpiry"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Expiry Date"
+            android:paddingTop="10dp"
+            android:paddingLeft="15dp"
+            android:textSize="16dp" />
```

Comment:
If we want these to be standard, extract them to the styles page, so you do not have to repeat for each text view, and only override when necessary.